src/compat/glibc_compat.cpp

```diff
@@ -0,0 +1,102 @@
+// Copyright (c) 2009-2018 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
```

Comment:
Are we required to supply their COPYING file?

Reply:
Not sure, but it still contains some of the lines they use as well.